### ISSUE TITLE
Feature request: Cell URLs

### ISSUE THREAD

**Issue by elbie-x** [HappenApps/Quiver#368]
Not sure if this is possible, but since cells are ordered and movable, could each cell be assigned a unique, hard URL so that it can be referred to in its own note/another note/another app? I can imagine this just being an extra string of random characters at the end of a note's URL, e.g., if my note link is quiver-note-url/21A42986-2FC6-4212-9BCD-F268198DB335, some arbitrary cell's link in there could be quiver-note-url/21A42986-2FC6-4212-9BCD-F268198DB335**/RNDCHR**. Would be great.

**Comment by ylian** [comment]
I have something else planned --- table of contents support. I think it's more useful than cell URLs.

Cells are meant to separate content types for easy editing, but they don't necessarily reflect the logical structure of a note. For example, you might have a section which has a heading in a text cell, then a code cell, then a text cell with some explanation of the code. The logical structure would refer to the three cells as a whole.

In that case it's better to link to the heading. This is the same as how GitHub links to rendered Markdown. Each `<h1>`, `<h2>`... becomes a link.

**Comment by drumanagh** [comment]
A table of contents sounds like a good idea. I'd personally find it super useful if it generates anchors that can be used by implicit reference links in Markdown, [Pandoc style](http://pandoc.org/README.html#extension-implicit_header_references) - e.g. `## Header title` becomes `<h2 id="header-title">Header title</h2>`, `[Header title]` becomes `<a href="#header-title">Header title</a>`, and `[this is a link][Header title]` becomes `<a href="#header-title">this is a link</a>` (assuming there isn't an explicitly defined "Header title" reference link to override the latter two).

**Comment by malkomalko** [comment]
I just want to add a comment here that being able to link to another notes specific cell would be very very handy.  There are a lot of times I find myself wanting to reference a particular section in a large note and right now you can only link to a specific note.

Or linking to headers would be great too!

**Comment by ChristophBoget** [comment]
Yes.  This.  Please!  It's been almost a year -- is this functionality in the pipeline at all?

**Comment by ChristophBoget** [comment]
And another year yet.
Bueller?
